proseco/guide.py

```diff
@@ -64,6 +64,8 @@ def get_guide_catalog(obsid=0, **kwargs):
 
 
 class GuideTable(ACACatalogTable):
+    # Catalog type when plotting (None | 'FID' | 'ACQ' | 'GUI')
```

Comment:
Here and elsewhere there are code changes that are part of #221, so I'm confused.  I just tried rebasing on master and that went smoothly, so you should do that.